swri_transform_util/include/swri_transform_util/local_xy_util.h

```diff
@@ -80,14 +80,25 @@ namespace swri_transform_util
       double& longitude);
 
   /**
-   * Utility class for converting between WGS84 lat/lon and an ortho-rectified
+   * @brief Utility class for converting between WGS84 lat/lon and an ortho-rectified
```

Comment:
There's a yaml config file for rosdoc. It might be possible to enable it there, but I don't know what options it supports.